schema/gen/go/testcase_test.go

```diff
@@ -0,0 +1,242 @@
+package gengo
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/polydawn/refmt/json"
+	"github.com/polydawn/refmt/shared"
+	. "github.com/warpfork/go-wish"
```

Comment:
I wonder if you've seen https://pkg.go.dev/github.com/frankban/quicktest. Its design is pretty interesting, and chunks of it are being upstreamed in the form of `testing.T.Cleanup` and `testing.T.TempDir`.

I'm not a fan of the dot-import, though. I get that the APIs stutter otherwise, like `wish.Wish(...)`, but that seems like a design issue that should be fixed and not worked around with dot-imports.

Reply:
I hadn't!  Haven't cachebusted my reviews of test libraries for a while.  That one looks nice.  Worth considering.